backend/prompt_studio/prompt_studio_core/views.py

```diff
@@ -393,51 +393,43 @@ def fetch_contents_ide(self, request: HttpRequest, pk: Any = None) -> Response:
         file_name: str = document.document_name
         view_type: str = serializer.validated_data.get("view_type")
 
-        # Extract filename without extension
         filename_without_extension = file_name.rsplit(".", 1)[0]
-
-        # Handle view_type logic, always converting to .txt for EXTRACT and SUMMARIZE
         if view_type == FileViewTypes.EXTRACT:
             file_name = (
                 f"{FileViewTypes.EXTRACT.lower()}/" f"{filename_without_extension}.txt"
             )
-        elif view_type == FileViewTypes.SUMMARIZE:
+        if view_type == FileViewTypes.SUMMARIZE:
```

Comment:
@jagadeeswaran-zipstack why was this reverted?

Reply:
These changes are in V1 which is deprecated. So reverted intentionally.